src/components/accounts/CreateAccount.js

```diff
@@ -67,6 +66,18 @@ class CreateAccount extends Component {
         token: ''
     }
 
+    componentDidMount = () => {
+        const { loginError, resetAccounts } = this.props;
+
+        if (loginError) {
+            // console.error('Error loading account:', loginError)
+
+            if (loginError.indexOf('does not exist while viewing') !== -1) {
```

Comment:
seems like this should be within Redux action / wallet. Doesn't really belong to UI component.

Also I wonder when exactly `loginError` does happen? Note that in some cases when transaction doesn't land for a long time we can end up with this error while account is still being created.